test/request_cache/plug_test.exs

```diff
@@ -0,0 +1,31 @@
+defmodule RequestCache.PlugTest do
+  @moduledoc false
+
+  use ExUnit.Case, async: true
+
+  defmodule FailingCacheModule do
+    def get(_cache_key) do
+      {:error, %{reason: :timeout}}
+    end
+  end
+
+  setup do
+    Application.put_env(:request_cache_plug, :request_cache_module, RequestCache.ConCacheStore)
+  end
+
+  describe "call/2" do
+    setup do
+      Application.put_env(:request_cache_plug, :request_cache_module, FailingCacheModule)
+    end
+
+    test "it handles errors from the cache implementation on GraphQL endpoints" do
+      conn = %Plug.Conn{request_path: "/graphql", query_string: "?query=query MyQuery{myQuery{}}", method: "GET"}
+      assert %Plug.Conn{} = RequestCache.Plug.call(conn, nil)
+    end
+
+    test "it handles errors from the cache implementation on REST endpoints" do
+      conn = %Plug.Conn{request_path: "/my/object/1", query_string: "?page=1", method: "GET"}
+      assert %Plug.Conn{} = RequestCache.Plug.call(conn, nil)
+    end
```

Comment:
should we use `capture_log` to verify the logging? probably not necessary to be honest

Reply:
:+1: